__tests__/index.spec.js

```diff
@@ -320,3 +320,50 @@ describe('using ref to the ReactTextareaAutocomplete to call methods', () => {
     expect(actual).toBe(expected);
   });
 });
+
+describe('data provided with keys', () => {
+  const mockedChangeFn = jest.fn();
+  const mockedSelectFn = jest.fn();
+  const mockedCaretPositionChangeFn = jest.fn();
+
+  const createRtaComponent = override => (
```

Comment:
I would probably remove the override as soon as we don't use it. What do you think?

Reply:
I was trying to mock `console.error` and use `override` to pass data without keys, but I couldn't get it working. 
If you don't include keys and have a duplicate output, react throws an warning about duplicate keys to `console.error`. The test output is missing this, so we're 👍